_templates/header.php

```diff
@@ -137,10 +137,10 @@
                     <li><a href="https://blog.elementary.io" target="_self">Blog</a></li>
                 </ul>
                 <ul class="right">
-                    <li><a href="https://youtube.com/user/elementaryproject" target="_blank" rel="noopener" data-l10n-off title="Youtube"><i class="fab fa-youtube"></i></a></li>
-                    <li><a href="https://mastodon.social/@elementary" target="_blank" rel="noopener me" data-l10n-off title="Mastodon"><i class="fab fa-mastodon"></i></a></li>
-                    <li><a href="https://www.reddit.com/r/elementaryos" target="_blank" rel="noopener" data-l10n-off title="Reddit"><i class="fab fa-reddit"></i></a></li>
-                    <li><a href="https://community-slack.elementary.io/" target="_blank" rel="noopener" data-l10n-off title="Slack"><i class="fab fa-slack"></i></a></li>
+                    <li><a href="https://youtube.com/user/elementaryproject" target="_blank" rel="noopener" data-l10n-off aria-label="Youtube" title="Youtube"><i aria-hidden="true" class="fab fa-youtube"></i></a></li>
```

Comment:
Isn't it usually "YouTube"? My understanding is that you want to capitalize everyone that you want pronounced as a separate word.

Reply:
good catch!